src/components/HelpScoutBeacon/suggestions.json

```diff
@@ -0,0 +1,28 @@
+[
+  [
+    "home",
+    [
+      "5cce35302c7d3a177d6e5b78",
+      "5cce3c4c2c7d3a177d6e5ba2",
+      "5cce34a32c7d3a177d6e5b74",
+      "5cce259a2c7d3a177d6e5b33",
+      "5c98a9330428633d2cf3fd23"
+    ]
+  ],
+  ["settings", ["5cce3bd204286306738eb793", "5c98a9330428633d2cf3fd23"]],
+  ["apps", ["5cce3baf04286306738eb792", "5c98a9330428633d2cf3fd23"]],
+  ["permissions", ["5cce3a7104286306738eb788", "5c98a9330428633d2cf3fd23"]],
+  [
+    "onboarding",
+    [
+      "5cce262f2c7d3a177d6e5b36",
+      "5cce28e82c7d3a177d6e5b40",
+      "5cce29a72c7d3a177d6e5b46",
+      "5cce2f0f2c7d3a177d6e5b5a",
+      "5c98a9330428633d2cf3fd23"
+    ]
+  ],
+  ["finance", ["5cce39b32c7d3a177d6e5b8b", "5c98a9330428633d2cf3fd23"]],
+  ["tokens", ["5cce38892c7d3a177d6e5b84", "5c98a9330428633d2cf3fd23"]],
```

Comment:
This is another reason why we should use the appId here instead of the name. Changing the appID is much much harder than the name 😄.

Reply:
Done in https://github.com/aragon/aragon/pull/853.